docs/guides/mlp_tutorials/llm-fine-tuning.md

```diff
@@ -8,45 +8,49 @@ This means that we take the model and train it on some new custom data to change
 To complete the tutorial, we set up some extra libraries that will help us to update the state of the machine learning model.
 We also write a script that will allow us to unlock more of the performance offered by the cluster, by running our fine-tuning task on two or more nodes.
 
+## Fine-tuning Gemma 7B on the OpenAssistant dataset
+
 ### Prerequisites
 
 This tutorial assumes you've already successfully completed the [LLM Inference][ref-mlp-llm-inference-tutorial] tutorial.
-For fine-tuning Gemma, we will rely on the NGC PyTorch container and the libraries we've already installed in the Python environment used previously.
+For fine-tuning Gemma, we will rely on the NGC PyTorch container and the libraries we've already installed in the Python virtual environment used previously.
 
 ### Set up TRL
 
-We will use HuggingFace TRL to fine-tune Gemma-7B on the [OpenAssistant dataset](https://huggingface.co/datasets/OpenAssistant/oasst_top1_2023-08-25).
+We will use HuggingFace TRL (Transformer Reinforcement Learning) to fine-tune Gemma-7B on the [OpenAssistant dataset](https://huggingface.co/datasets/OpenAssistant/oasst_top1_2023-08-25).
 First, we need to update our Python environment with some extra libraries to support TRL.
 To do this, we can launch an interactive shell in the PyTorch container, just like we did in the previous tutorial.
 Then, we install `peft`:
 
-```console
-$ cd $SCRATCH/gemma-inference
-$ srun --environment=gemma-pytorch --container-workdir=$PWD --pty bash
-$ source ./gemma-venv/bin/activate
-$ python -m pip install peft==0.11.1
+```bash
+$ cd $SCRATCH/tutorials/gemma-7b
+$ srun --environment=./ngc-pytorch-gemma-24.01.toml --pty bash
+$ source venv-gemma-24.01/bin/activate
+$ pip install peft==0.11.1
```

Comment:
```suggestion
cd $SCRATCH/tutorials/gemma-7b
srun --environment=./ngc-pytorch-gemma-24.01.toml --pty bash
source venv-gemma-24.01/bin/activate
pip install peft==0.11.1
```
Since it was changed to `bash`.